build/workflow.js

```diff
@@ -13,9 +13,11 @@ const yaml  = require('js-yaml')
 // === Constants ===
 // =================
 
-const NODE_VERSION      = '14.15.0'
-const RUST_VERSION      = 'nightly-2019-11-04'
-const WASM_PACK_VERSION = '0.9.1'
+const NODE_VERSION             = '14.15.0'
```

Comment:
Are we not getting this version programmatically somewhere?

Reply:
Good question and good observation. Yes, we repeat things in CI, we always did. I'll add a task to fix it, but it will take some good refactoring one day.